tests/features/community_content/notification.post_moderated.discussion.feature

```diff
@@ -86,7 +86,7 @@ Feature: Notification test for the discussion transitions on a post moderated pa
     Then the following email should have been sent:
       | recipient | CC member                                                                                                                                                        |
       | subject   | Joinup: Content has been updated                                                                                                                                 |
-      | body      | the Facilitator, CC Facilitator has approved your request to publish the discussion - "CC notify post approve proposed" in the collection: "CC post collection". |
+      | body      | the Facilitator, CC Facilitator has approved your request of publication of the discussion - "CC notify post approve proposed" in the collection: "CC post collection". |
```

Comment:
I think you misunderstand me. This is really poor language and formatting.

> the

This sentence doesn't start with a capital letter.

> Facilitator,

In English nouns are not capitalized like they are for example in German. The comma here is misplaced.

> CC Facilitator

Here a comma is missing.

> has approved your request of publication of the discussion

This just sounds completely wrong.

> - "CC [...]"

What is this hyphen doing here?

> in the collection: "CC [...]"

What is this double colon doing here?